server/src/services/job.service.ts

```diff
@@ -162,7 +162,7 @@ export class JobService extends BaseService {
       }
 
       case QueueName.LIBRARY: {
-        return this.jobRepository.queue({ name: JobName.LIBRARY_QUEUE_SYNC_ALL, data: { force } });
+        return this.jobRepository.queue({ name: JobName.LIBRARY_QUEUE_SCAN_ALL, data: { force } });
```

Comment:
We already have an API that queues this job as you can see with this line. We also have another endpoint specifically for manually creating jobs (in the job controller). Can you please add the functionality via one of those two existing methods?

Reply:
Good catch, I missed that.

I'll rework the PR.